### PR TITLE
Clarity, Consistency, and Prose in Damage Types

### PR DESCRIPTION
**Pull request by Turtlelord26**
Cleaning up, adding prose, and template consistency. Added explicit defensive rules to all damage types, and restyled BPL into damage types instead of weapon types, which really doesn't change anything but is more consistent. That last bit is from #492.

New Issue, but Fall damage really could move elsewhere, since nothing *deals* fall damage, characters only suffer it.
Also New Issue: the sections on shield and armor are redundant if these changes go through, and can be deleted.

One functional change: Added shield bypass property to Acid damage, since shields probably only block high-energy low-mass forms of attack.

### CODE REVIEW

File: 01_04_Damage_Types_and_Armor_Types.txt [hunk 1]
```diff
@@ -4,81 +4,89 @@ Damage Types and Armor Types
...
+they gain one instance of Plasma Burn (see Conditions). 
+
+	Armor damaged by Plasma loses 1 APL. This effect cannot stack and wears off 
+after the encounter.
+
+Laser Damage:
+	Laser damage is a burn or explosive energy release delivered by a high-
+intensity stream of photons, or light.
+	
+	Laser damage is dealth first to shields, and then to health. It
```
Comment (rosebrallier) [inline 01_04_Damage_Types_and_Armor_Types.txt]: dealt

File: 01_04_Damage_Types_and_Armor_Types.txt [hunk 2]
```diff
@@ -4,81 +4,89 @@ Damage Types and Armor Types
...
-
-Explosives:
-    Certain grenades and devices deal Explosive damage, which ignores armor and 
-shields (explosive damage is essentially a crushing force). Though they may seem 
+== Common Damage Types ==
+
+Ballistic Damage:
+	Ballistic damage results from the impact and/or penetration of solid 
+projectiles, causing physical damage and secondary shock effects to living and 
+mechanical systems.
```
Comment (bleehu) [inline 01_04_Damage_Types_and_Armor_Types.txt:12]: Good prose! Solid improvement!

File: 01_04_Damage_Types_and_Armor_Types.txt [hunk 3]
```diff
@@ -4,81 +4,89 @@ Damage Types and Armor Types
...
+    
+	Ballistic damage is dealt first to armor, and then to health. It 
+bypasses shields.
+
+Plasma Damage:
+	Plasma damage is dealt by energy transfer to the target from a high-energy 
+plasma projectile and/or spray.
```
Comment (bleehu) [inline 01_04_Damage_Types_and_Armor_Types.txt]: nonblocking: this _may_ be a little technical, and doesn't add much flavor. While you and I had thermodynamics courses in college and we understand the damage dealt by heat and EM energy transfer, this doesn't really make any more of an explination than "plasma damage happens when you hit things with plasma". It's still better than having no prose whatsoever.

File: 01_04_Damage_Types_and_Armor_Types.txt [hunk 4]
```diff
@@ -4,81 +4,89 @@ Damage Types and Armor Types
...
+	Plasma damage is dealt first to shields, then to armor, and then to health, 
+but shields and armor can only absorb up to half of each plasma attack's damage 
+(rounded up). A character with armor but no shields (and no other defensive effects) 
+would always take at least half of a plasma attack's damage to their health.
+
+	Additionally, whenever a character takes Plasma damage to their health, 
+they gain one instance of Plasma Burn (see Conditions). 
+
+	Armor damaged by Plasma loses 1 APL. This effect cannot stack and wears off 
+after the encounter.
```
Comment (bleehu) [inline 01_04_Damage_Types_and_Armor_Types.txt:31]: Breaking this up into separate paragraphs makes this much easier to skim. Thanks for doing this!

File: 01_04_Damage_Types_and_Armor_Types.txt (hunk 1 above)
Comment (bleehu) [inline 01_04_Damage_Types_and_Armor_Types.txt]: [SP] "dealth"

**Review by bleehu — APPROVED** [review]
One spelling correction, otherwise fantastic work!

File: 01_04_Damage_Types_and_Armor_Types.txt [hunk 5]
```diff
@@ -4,81 +4,89 @@ Damage Types and Armor Types
...
+	
+	Laser damage is dealth first to shields, and then to health. It 
+bypasses armor.
+
+== Miscellaneous Damage Types ==
+
+Explosive Damage:
+	Certain grenades and devices deal Explosive damage, which is a shockwave 
+transmitted through any liquid or gaseous medium. Though they may seem 
 like explosive weapons, fragmentation grenades deal ballistic damage because their
```
Comment (rosebrallier) [inline 01_04_Damage_Types_and_Armor_Types.txt]: This sentence might read better with the "they" and "fragmentation grenades" swapped.

**Review by rosebrallier — APPROVED** [review]
THANK YOU SO MUCH THIS HAS BEEN DRIVING ME NUTS

**Pull request merged**